operators/vtk_renderer/vtk_renderer.cpp

```diff
@@ -0,0 +1,189 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2024 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "vtk_renderer.hpp"
+
+#include "holoscan/core/execution_context.hpp"
+#include "holoscan/core/gxf/entity.hpp"
+#include "holoscan/core/io_context.hpp"
+#include "holoscan/core/operator_spec.hpp"
+#include "holoscan/core/resources/gxf/cuda_stream_pool.hpp"
```

Comment:
Is CudaStreamPool used? I could not find it.

Reply:
Not used, I removed it